core/client/kt/verify.go

```diff
@@ -25,15 +25,16 @@ import (
 	"github.com/google/keytransparency/core/crypto/commitments"
 	"github.com/google/keytransparency/core/crypto/vrf"
 	"github.com/google/keytransparency/core/tree/sparse"
-	tv "github.com/google/keytransparency/core/tree/sparse/verifier"
```

Comment:
Please keep this here, AFAIK we're putting all Key Transparency imports in the same group regardless of they're renamed.